### PR TITLE
Neutron HA changes

### PR DESCRIPTION
**Pull request by rahulait**

### CODE REVIEW

**Comment by radonm** [comment]
+1

File: quickstack/manifests/neutron/controller.pp [hunk 1]
```diff
@@ -348,6 +358,7 @@
     cert_file             => $cert_file,
     key_file              => $key_file,
     ca_file               => $ca_file,
+    dhcp_agents_per_network => $dhcp_agents_per_network,
```
Comment (kamfonik) [inline quickstack/manifests/neutron/controller.pp:361]: hash arrows are not aligned. ;-)

**Comment by kamfonik** [comment]
looks good except for the alignment which could be fixed later. +1

**Comment by rahulait** [comment]
@kamfonik Done. :)

**Pull request merged**